lib/node/http.js

```diff
@@ -72,6 +79,22 @@ class HTTP extends Server {
 
     this.initRouter();
     this.initSockets();
+    this.initS3();
+  }
+
+  /**
+   * Initialize the S3 service, if possible.
+   * @private
+   */
+
+  initS3() {
+    AWS.config.getCredentials((err) => {
```

Comment:
Are these credentials from the instance?

Reply:
Yes, these are credentials from any of the standard methods AWS tries to discover credentials. On a dev machine, this could be from ~/.aws or environment variables, on an instance it will just work.